README.rst

```diff
@@ -1,39 +1,67 @@
 .. SPDX-License-Identifier: MIT OR Apache-2.0
    SPDX-FileCopyrightText: The Ferrocene Developers
+   SPDX-FileCopyrightText: The Rust Project Contributors
 
 ================================
-Ferrocene Language Specification
+FLS
 ================================
 
 .. raw:: html
 
-   <p align="center"><a href="https://spec.ferrocene.dev">Read the
-   specification &raquo;</a></p>
+   <p align="center"><a href="https://rust-lang.github.io/fls">Read the document
+   &raquo;</a></p>
 
-The Ferrocene Language Specification (FLS) is a document describing the Rust
-language. It was created as one of the prerequisites for qualifying
-`Ferrocene`_, a Rust toolchain qualified for safety-critical environments. The
-FLS is compiled of existing Rust documentation, but presented with a rigorous
-structure in order to meet the requirements of qualification.
+The FLS is a document describing aspects of the Rust language for Rust toolchain
+qualification purposes.
 
-The FLS is not intended to be used as the normative specification of the Rust
-language, nor is it meant to replace the decision-making processes of the Rust
-project. Any difference between the FLS and the behavior of the Rust compiler
-is considered an error on our part and the FLS will be updated accordingly.
+It was created by Ferrous Systems, in an original joint effort with AdaCore, as
+one of the prerequisites for qualifying `Ferrocene`_, a Rust toolchain qualified
+for safety-critical environments. The FLS is compiled of existing Rust
+documentation, but presented with a rigorous structure in order to meet the
+requirements of qualification.
 
-The Ferrocene Language Specification text is licensed under either the ``MIT``
-or ``Apache-2.0`` licenses, at your option. Individual files might have
-different licensing. Licensing metadata is present in each file, and the full
-licenses text is present in the ``LICENSES/`` directory.
+The FLS is not intended to be used as *the* normative specification of the Rust
+language (see the `Rust Reference`_), nor is it meant to replace the
+decision-making processes of the Rust project. Any difference between the FLS
+and the behavior of the Rust compiler is considered an error on our part and the
+FLS will be updated accordingly.
+
+The FLS text is licensed under either the ``MIT`` or ``Apache-2.0`` licenses, at
+your option. Individual files might have different licensing. Licensing metadata
+is present in each file, and the full licenses text is present in the
+``LICENSES/`` directory.
 
 .. _Ferrocene: https://ferrocene.dev
+.. _Rust Reference: https://doc.rust-lang.org/reference/
+
+Acknowledging Ferrous Systems
+===============================
```

Comment:
The underline length needs fixing.  I'll do it.  I'm making a final pass and will merge.